### PR TITLE
Add more wrappers and a pkg-config file

### PR DESCRIPTION
**Pull request by myfreeweb**
This is for my [capsicumizer](https://github.com/myfreeweb/capsicumizer), which can now run gedit :)

### CODE REVIEW

File: lib/po_libc_wrappers.c [hunk 1]
```diff
@@ -199,6 +200,26 @@ open(const char *path, int flags, ...)
...
 
+/**
+ * Capability-safe wrapper around the `rename(2)` system call.
+ *
+ * `rename(2)` accepts a path argument that can reference the global filesystem
+ * namespace. This is not a capability-safe operation, so this wrapper function
+ * attempts to look up the path (or a prefix of it) within the current global
+ * po_map and converts the call into the capability-safe `renameat(2)` if
+ * possible. If the current po_map does not contain the sought-after path,
+ * this wrapper will call `renameat(AT_FDCWD, original_path, ...)`, which is
```
Comment (emaste) [inline lib/po_libc_wrappers.c]: i.e., we'll return `ECAPMODE` _Not permitted in capability mode_ if we do not have a suitable path in `po_map`. Sounds good.

File: lib/po_libc_wrappers.c [hunk 2]
```diff
@@ -218,6 +218,25 @@ stat(const char *path, struct stat *st)
...
+ * possible. If the current po_map does not contain the sought-after path,
+ * this wrapper will call `fstatat(AT_FDCWD, original_path, ...)`, which is
+ * the same as the unwrapped `lstat(2)` call.
+ */
+int
+lstat(const char *path, struct stat *st)
+{
+	struct po_relpath rel = find_relative(path, NULL);
+
+	return fstatat(rel.dirfd, rel.relative_path,st,AT_SYMLINK_NOFOLLOW);
```
Comment (emaste) [inline lib/po_libc_wrappers.c]: space after ,

**Comment by emaste** [comment]
Overall patch looks reasonable to me. I'll tag @oshogbo also to take a look.

**Comment by trombonehero** [comment]
Looks good... thanks very much! I can touch up the style issues that @emaste raised post-merge.

**Pull request merged**